### PR TITLE
[OBMIN-96] Added the ability to delete an account, updated some files

### PR DESCRIPTION
**Pull request by cloudmenson**

### CODE REVIEW

File: src/main/resources/react/src/REST/Resources/profile.js [hunk 1]
```diff
@@ -26,3 +26,9 @@ export const putAvatar = (avatar) =>
 
 export const deleteAvatar = () =>
   obminyashkaApi.delete('/user/service/avatar').then(({ data }) => data);
+
+export const deleteUserAccount = (userData) => {
+  return obminyashkaApi
```
Comment (Wolshebnik) [inline src/main/resources/react/src/REST/Resources/profile.js:31]: do it how above

File: src/main/resources/react/src/Utils/validation.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import * as yup from 'yup';
+
+import { getTranslatedText } from 'components/local/localization';
+
```
Comment (Wolshebnik) [inline src/main/resources/react/src/Utils/validation.js:4]: delete space

File: src/main/resources/react/src/pages/UserInfo/tab_pages/my_activity/index.jsx [hunk 3]
```diff
@@ -3,14 +3,19 @@ import { TitleBigBlue } from 'components/common';
...
 import { CardBlock, Container, StyledCardBlock } from './styles';
 
 const MyActivity = () => {
   const isFavorite = true;
 
   return (
     <Container>
-      <TitleBigBlue whatClass="incoming__replies-text" text="Входящие ответы" />
+      <TitleBigBlue
+        whatClass="incoming__replies-text"
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/UserInfo/tab_pages/my_activity/index.jsx:16]: maybe this class can be delete

File: src/main/resources/react/src/pages/UserInfo/tab_pages/my_activity/index.jsx [hunk 4]
```diff
@@ -60,8 +65,8 @@ const MyActivity = () => {
       </CardBlock>
 
       <TitleBigBlue
-        text="Исходящие ответы"
         whatClass="outgoing__replies-text"
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/UserInfo/tab_pages/my_activity/index.jsx:68]: the same

File: src/main/resources/react/src/pages/UserInfo/tab_pages/my_settings/index.jsx [hunk 5]
```diff
@@ -1,23 +1,36 @@
...
+} from './config';
 
 import * as Styles from './styles';
 
 const MySettings = () => {
   const dispatch = useDispatch();
   const { openModal } = useContext(ModalContext);
 
+  const [loading, setLoading] = useState(false);
+
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/UserInfo/tab_pages/my_settings/index.jsx:33]: delete space pls

File: src/main/resources/react/src/REST/Resources/profile.js (hunk 1 above)
Comment (Wolshebnik) [inline src/main/resources/react/src/REST/Resources/profile.js:31]: do it how above pls

File: src/main/resources/react/src/REST/Resources/profile.js [hunk 6]
```diff
@@ -26,3 +26,9 @@ export const putAvatar = (avatar) =>
...
+    .delete('/user/service/delete', { data: userData })
+    .then(({ data }) => data);
```
Comment (Wolshebnik) [inline src/main/resources/react/src/REST/Resources/profile.js:33]: Are you sure it(then) needed?

File: src/main/resources/react/src/pages/UserInfo/tab_pages/my_activity/index.jsx [hunk 7]
```diff
@@ -3,14 +3,19 @@ import { TitleBigBlue } from 'components/common';
 import { ProductCard } from 'components/item-card';
 import mommy from 'assets/img/mama_1_04232631 1.png';
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/UserInfo/tab_pages/my_activity/index.jsx:5]: delete space pls

**Review by smilaya — APPROVED** [review]

**Review by Wolshebnik — APPROVED** [review]

**Pull request merged**